src/components/DataTable/README.md

```diff
@@ -174,6 +174,34 @@ function DataTableFooterExample() {
 }
 ```
 
+### Data table with custom totals labels
+
+Use to pass in custom totals labels for both singular and plural amount of totals items. The plurality is based on the amount of items passed in for totals.
+
+```jsx
+function DataTableExample() {
+  const rows = [['Emerald Silk Gown', '$875.00']];
+
+  return (
+    <Page title="Sales by product">
+      <Card>
+        <DataTable
+          columnContentTypes={['text', 'text']}
+          headings={['Products']}
+          rows={rows}
+          totals={['', '$155,830.00']}
+          totalsMarkupLabel={{
+            singular: 'Singular',
+            plural: 'Plural',
```

Comment:
Is there a more realistic example that we can use here? Maybe from the use case that prompted these changes?

Reply:
Good idea, changing it to `Total` and `Totals`